.build_web.py

```diff
@@ -31,6 +34,13 @@ def build_web():
                 os.mkdir("../dist")
 
             copyfile("build/web_assets.h", "../dist/web_assets.h")
+
+            generated_timestamp = "//-"+str(int(time()))+"-"
+            with open("../dist/web_assets.h", 'r+') as f:
+                content = f.read()
+                f.seek(0, 0)
+                f.write(generated_timestamp + '\n' + content)
+                
```

Comment:
Could we not just compare `st_mtime` of the `web_assets` file from `os.stat` instead of dumping/parsing it?

Reply:
I think I had a reason for this, but I can't remember to be perfectly honest. 3am code isn't the most logical at times. :sweat_smile: 

I'll swap it to just the file's `st_mtime` unless either of us can find a reason to keep it in.

Besides, the script should be cleaner without it.

Reply:
Awesome, thanks :)